### PR TITLE
Like and match history sprint 7

### PR DESCRIPTION
**Pull request by marluxiaboss**
PR for the two week sprint.

### CODE REVIEW

File: app/src/androidTest/java/ch/epfl/sdp/blindwar/ProfileActivityTest.kt [hunk 1]
```diff
@@ -0,0 +1,160 @@
+/*
```
Comment (ghemDD) [inline app/src/androidTest/java/ch/epfl/sdp/blindwar/ProfileActivityTest.kt:1]: Remove file

**Review by ghemDD — APPROVED** [review]
Nice work 
Maybe we can use a more general recycler view to display both songs and playlists as to avoid duplication

File: app/src/main/java/ch/epfl/sdp/blindwar/profile/model/User.kt [hunk 2]
```diff
@@ -44,8 +49,10 @@ data class User(
         private var pseudo: String = "",
         private var firstName: String = "",
         private var lastName: String = "",
-        private var birthdate: Long = -1,
+        private var birthDate: Long = -1,
```
Comment (CHOCOLVT) [inline app/src/main/java/ch/epfl/sdp/blindwar/profile/model/User.kt:52]: should have been let to birthdate

**Review by CHOCOLVT — APPROVED** [review]
should be more careful when merging from main, avoiding keeping old stuff / files

but otherwise the added code is good

**Pull request merged**